pages/login.tsx

```diff
@@ -0,0 +1,122 @@
+import Image from "next/image";
+import React, { ChangeEvent, useState } from "react";
+import { login as translate } from "../public/locales/fa.json";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+
+const Login = () => {
+  const [name, setName] = useState<string>();
+  const [password, setPassword] = useState<string>("");
+
+  const usernamechangehandler = (event: ChangeEvent<HTMLInputElement>) => {
```

Comment:
- Check out naming conventions for functions
```suggestion
  const usernameChangeHandler = (event: ChangeEvent<HTMLInputElement>) => {
```